topiary/src/error.rs

```diff
@@ -207,3 +207,12 @@ impl From<tree_sitter_facade::ParserError> for FormatterError {
         Self::Internal("Error while parsing".into(), Some(Box::new(e)))
     }
 }
+
+impl From<toml::de::Error> for FormatterError {
+    fn from(e: toml::de::Error) -> Self {
+        Self::Internal(
+            "Error while parsing the internal configuration file".to_owned(),
```

Comment:
I think `internal` is misleading here. Unless I'm mistaken, the error can also happen when reading a user-defined config file.

Reply:
Hej! The library is only responsible for parsing the default configuration, and such that error can only occur then. Parsing the user configuration is done in topiary-cli, and that error thus does have a different message: https://github.com/tweag/topiary/pull/506/files#diff-c77596c57513055d5393df388690627ffe805f456ccbed4f98e48d83304ea219R123

Reply:
Having said that, at your discretion, it might be better to just .expect() after parsing the internal configuration file, this error should not occur anyway.

Reply:
I just misunderstood, it's okay to leave it as it is.